src/extension/migration.js

```diff
@@ -0,0 +1,41 @@
+export default function migrateStyle(properties) {
+  const dataTemplateLabel = {
+    label: {
+      value: {
+        colorType: undefined,
```

Comment:
Why are these properties here if they are `undefined`?

Reply:
I've made some changes to migration to fix all the problems mentioned above.